CMakeLists.txt

```diff
@@ -299,10 +299,18 @@ set_target_properties(lib${APP_NAME_LC} PROPERTIES PREFIX "")
 set(OTHER_FILES cmake/README.md)
 
 # Subdirs
+# Tempo hack
+if(CORE_PLATFORM_NAME STREQUAL "tvos")
```

Comment:
As you already have noticed yourself, this hack is wrong.

Reply:
This is still a hack.

Reply:
Hum yes. But this is an artefact of using the same `CORE_SYSTEM_NAME ` value with tvOS and iOS.
Do you think about another approach in order to use the correct `subduers.txt` file?

Reply:
What do you think of using the same approach as rbpi? I mean:
* Set `CORE_PLATFORM_NAME` to `ios` or `tvos` depending on the choose target
* Put in `cmake/treedata/ios/subdirs.txt` only folders/files that are common to iOS and tvOS
* Use the `if(CORE_PLATFORM_NAME_LC STREQUAL xxxx)  add_subdirectory(xxxx) endif()` approach in `CmakeLists.txt` files that are in `xbmc/windowing` or `xbmc/platform` for files that are not the same for IOS and tvOS.

Then we do not need to modify the main `CmakeLists.txt` file.

Reply:
- Yes
- Yes
- No, that's wrong. Adapt cmake to read `cmake/treedata/ios/${CORE_PLATFORM_NAME_LC}/` directory and add the distinct directories there.